test/functional/wiki_controller_test.rb

```diff
@@ -434,7 +434,8 @@ def teardown
     UserSession.create(users(:jeff))
     get :show, params: { id: 'A-new-wiki-page' }
     assert_response :success
-    assert_template 'wiki/edit'
+    assert_template 'editor/wikiRich'
```

Comment:
This looks right, and we should be able to remove all the commented lines now. 

Finally, the last error is probably due to the new template having a different tag input form. We should ensure that the test looks for a tag input form that's really there -- it's not finding one with id `taginput` and value of `one,two`. Hope that helps!

```
test_should_use_existing_node_body_as_template_in_post_form_based_on_param_‘n’(Minitest::Result) [/app/test/functional/wiki_controller_test.rb:51]: Expected at least 1 element matching “#taginput[value=”one,two”]”, found 0.. Expected 0 to be >= 1.
```